python/ray/_private/node.py

```diff
@@ -1058,28 +1061,36 @@ def start_head_processes(self):
         assert self._gcs_client is not None
         self._write_cluster_info_to_kv()
 
-        if not self._ray_params.no_monitor:
-            self.start_monitor()
+        if not ray_in_tee():
+            # Workaround because these params are not available in ray.init().
+            # Should be reverted once we start Ray with `ray start` CLI.
```

Comment:
Why these should be reverted when we use `ray start`?

Reply:
> Is it possible to add a unit test for running ray in Occlum?

Data transmission is done by passing a physical mem address in Occlum, this behavior will incur crashes in many stages, e.g. `ray.init`, outside the Occlum container.
Therefore, no particular interesting tests.